lib/app/ui/screens/home/home_page.dart

```diff
@@ -34,8 +34,10 @@ class _HomePageState extends State<HomePage> {
         actions: [
           IconButton(
             icon: const Icon(Icons.logout),
-            onPressed: () =>
-                Provider.of<UserViewModel>(context, listen: false).signOut(),
+            onPressed: () {
+              Provider.of<UserViewModel>(context, listen: false).signOut();
+              Navigator.of(context).pushNamed('/login');
```

Comment:
Looking good, but I'm considering having a state manager handle these types of logic (maybe for a future version).
e.g. If we're signed out then redirect to /login.

But for now, this will do. Great work!

Reply:
That's true, we can just have a wrapper around the `main.dart`, great idea 👍🏽